app/src/main/java/com/github/h3lp3rs/h3lp/storage/Storages.kt

```diff
@@ -7,11 +7,11 @@ import java.lang.Boolean.parseBoolean
 /**
  * Enumeration of all useful (local) storages in H3LP
  */
-enum class Storages {
-    USER_COOKIE, MEDICAL_INFO, SKILLS, EMERGENCIES_RECEIVED, FORUM_THEMES_NOTIFICATIONS, FORUM_CACHE;
+enum class Storages(private var isFresh: Boolean) {
+    USER_COOKIE(false), MEDICAL_INFO(false), SKILLS(false), EMERGENCIES_RECEIVED(false), FORUM_THEMES_NOTIFICATIONS(false), FORUM_CACHE(false), SIGN_IN(true);
 
     private val ls = LocalStorage(name, getGlobalCtx())
-    private var isFresh = false
+    //private var isFresh = false
```

Comment:
It seems like a massive change, why?

When we implemented the storage, we used this variable so that we always pull the very first time a storage is used and therefore there is no need to know in the code if the data is freshed or not, if the storage has been used somewhere else then it's freshed otherwise it's not. I don't understand what is the problem of pulling in your case (ie. having isFresh false)

Reply:
Well it's not a massive change, all the other storages are kept "not fresh" at the beginning. The thing is that the purpose of using a storage SIGNIN is to allow a completely offline mode with a sign-in option. Which means not using the remote database at all. That's why I also disabled online synchronization. Also, the first call to pull needs a user ID, which we don't have yet since the user is not signed in yet (So it was actually impossible to keep the pull) this is not a problem for the other storages since they are in a context where we already have a user Id. Another reason is that I only store the username and the user Id, and this information is supposed not to change unless another user is signed in, thus, I feel like there is no need for checking if they conform to a remote database.

I am aware it was not the best solution to make isFresh true for the SIGNIN storage, but at this point I feel like it's the simplest way to do it. We can still change that in a future task if the time allows it😉

Reply:
Hehe yes thank you for the explanation. After my PR the onlineSync will be false by default (a lot of storages are onlineSynced but shouldn't be) so this won't be necessary, but I can make the change when I merge after you

Reply:
Oh cool that you did the change on your PR  👍🏻